docs/source/user-guide/tutorials/build-pkgs-skeleton.rst

```diff
@@ -129,15 +128,15 @@ environment by using the use-local flag:
 
 .. code-block:: bash
 
-    conda install --use-local pyinstrument
+    conda install --use-local click
```

Comment:
Add note here to notice click coming from local conda-bld channel opposed to pkg/main. Maybe a screenshot/codeblock would be nice too

Reply:
`(click) 0561:~ jsmith$ conda list
   # packages in environment at /Users/Jsmith/miniconda/envs/click:`

Something like this?